src/docs/product/security/mobile-privacy.mdx

```diff
@@ -54,3 +54,12 @@ No. Sentry does not cause your app to contain any ads.
 ### What do I need to disclose to Google Play?
 
 You would need to disclose and/or include in your app privacy policy all types of data you are collecting through your app, including data you are sending to Sentry.
+
+## Sentry SDKs for third-party SDKs and libraries
```

Comment:
I really like that we're documenting this! 👌 

I have one small point of feedback though: I would differentiate between "SDK use inside libraries" and "third-party SDKs" a bit more and separate the issues they introduce.

The main drawback of 3rd party SDKs is that they may not be as careful with PII, they will lag behind in feature support, and may not always support other tooling we have.

The main drawback of SDKs in libraries is, like you said, that multiple Sentry instances may conflict with each other, however, that is not a problem per se in 3rd party SDKs.

What do you think about splitting this up into two separate sections?

In general, this is very nice though so I won't block and I'll slap approve :)

Reply:
@lforst not sure I understand your point, SDKs, and libraries are pretty much a synonym here, if a 3rd party SDK or a 3rd party library embeds a  Sentry SDK in it, all the problems occur anyway.
Are you maybe referring to a Sentry SDK integration? which is not dependent on the core SDK itself? Or maybe something specific to the JS world?

Reply:
Well there are 3rd party SDKs that consume first party SDKs as dependencies. For example: https://github.com/nuxt-community/sentry-module

Libraries (unrelated to error monitoring or Sentry) that include Sentry SDKs are in a different class of problem because downstream users probably don't expect libraries to interfere with their Sentry setups. I think there is an inherent difference between third party SDKs (like the Nuxt SDK) and random libraries calling `Sentry.init()`.

Reply:
@lforst so if the `sentry-module` calls `Sentry.init` https://github.com/nuxt-community/sentry-module/blob/63d79061de0ac1edc2a933cb25b530c34f297e64/src/hooks.ts#L157
Someone adds `sentry-module` into their own app, they also add the Sentry JS SDK directly and also call `Sentry.init`, what's gonna be the behavior? where does the event go? the `sentry-module`s DSN or the App's DSN that called `Sentry.init` again?

Reply:
Anyway, since this section is Mobile only for now, we can tailor it to other platforms if needed, I don't think the `sentry-module` use case applies to Mobile in general.

Reply:
Most of this information except for the last bullet isn't relevant to Data Privacy for Mobile. I think it makes more sense to move this to the [platforms page](https://docs.sentry.io/platforms/), perhaps as a note before the second platforms heading listing all platforms.